contracts/core/Dispatcher.sol

```diff
@@ -160,9 +184,21 @@ contract Dispatcher is IbcDispatcher, IbcEventsEmitter, Ownable, Ibc {
             revert IBCErrors.channelNotOwnedBySender();
         }
 
-        IbcChannelReceiver reciever = IbcChannelReceiver(msg.sender);
-        reciever.onCloseIbcChannel(channelId, channel.counterpartyPortId, channel.counterpartyChannelId);
-        emit CloseIbcChannel(msg.sender, channelId);
+        IbcChannelReceiver receiver = IbcChannelReceiver(msg.sender);
+        (bool success, bytes memory data) = _try_catch(
```

Comment:
Is `data` used?

Reply:
Data either (in the case of a success) the returned value from the low-level call, or the bytes thrown by a revert in a low level call

Reply:
`data` is assigned but not used by the caller, right?   
This will be the same? 
```solidity
        (bool success,) = _try_catch(
```

Reply:
Actually this comment made me realize we can log error data in the error events, so I pushed some changes for that